### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by nath0127**

### CODE REVIEW

File: src/components/Dashboard/Dashboard.jsx [hunk 1]
```diff
@@ -90,27 +106,32 @@ const Dashboard = (props) => {
...
       <NavSidebar />
       <main className={clsx(classes.content, open && classes.contentShift)}>
-        <Container maxWidth="sm" className={classes.header}>
+        <Container maxWidth="md" className={classes.header}>
           <h1 className={classes.title}>Discover More Groups</h1>
           <Autocomplete
-            freeSolo
+            debug
+            noOptionsText={
+              <Button type="button" onClick={handleModalOpen}>
```
Comment (twheaton53) [inline src/components/Dashboard/Dashboard.jsx]:
```suggestion
              <Button type="button" onClick={() => setModalOpen(true)}>
```

File: src/components/Dashboard/Dashboard.jsx [hunk 2]
```diff
@@ -128,6 +149,14 @@ const Dashboard = (props) => {
             </Grid>
           </Container>
         </Container>
+        <Modal
+          open={modalOpen}
+          onClose={handleModalClose}
```
Comment (twheaton53) [inline src/components/Dashboard/Dashboard.jsx]:
```suggestion
          onClose={() => setModalOpen(false)}
```

File: src/components/Dashboard/Dashboard.jsx [hunk 3]
```diff
@@ -73,11 +83,17 @@ const categoryList = [
...
   const [open, setOpen] = useState(false);
   const [isLogged, setIsLogged] = useState([]);
   const classes = useStyles();
+  const [modalOpen, setModalOpen] = useState(false);
+  const handleModalOpen = () => {
+    setModalOpen(true);
+  };
+  const handleModalClose = () => {
+    setModalOpen(false);
+  };
```
Comment (twheaton53) [inline src/components/Dashboard/Dashboard.jsx]:
```suggestion
```

**Review by twheaton53 — CHANGES_REQUESTED** [review]

**Review by wackyleo459 — APPROVED** [review]
Looks good , seems no conflicts!

**Pull request merged**